cvat/apps/engine/views.py

```diff
@@ -309,19 +308,21 @@ def perform_create(self, serializer, **kwargs):
         )
 
     @extend_schema(
-        summary='Method returns information of the tasks of the project with the selected id',
-        responses=TaskReadSerializer(many=True)) # Duplicate to still get 'list' op. nam
+        summary='Moved to /tasks',
+        responses=TaskReadSerializer(many=True), # Duplicate to still get 'list' op. name
+        deprecated=True) # TODO: remove in v2.5
     @action(detail=True, methods=['GET'], serializer_class=TaskReadSerializer,
         pagination_class=viewsets.GenericViewSet.pagination_class,
+        # These non-root list endpoints do not suppose extra options, just the basic output
         # Remove regular list() parameters from the swagger schema.
         # Unset, they would be taken from the enclosing class, which is wrong.
         # https://drf-spectacular.readthedocs.io/en/latest/faq.html#my-action-is-erroneously-paginated-or-has-filter-parameters-that-i-do-not-want
-        filter_fields=None, search_fields=None, ordering_fields=None)
+        filter_fields=None, ordering_fields=None, search_fields=None, simple_filters=None)
     def tasks(self, request, pk):
```

Comment:
Let's remove all deprecated endpoints. They were changed (pagination). I see no reasons to keep them. In the future we will add {"projects": "link to list of tasks" }.

Reply:
Done